lib/payload.js

```diff
@@ -6,6 +6,8 @@
 // this file has no tests so avoid refactor and ignore jshint for now
 /*jshint ignore: start */
 var serial,__hasProp = {}.hasOwnProperty;
+var SG = require('strong-globalize');
```

Comment:
Why not following strong-globalize coding guideline here?
https://www.npmjs.com/package/strong-globalize#autonomous-message-loading
Please study paragraphs after "For example, the following does not work as intended because the package sub calls SG.SetRootDir first:"